.ci/set-milestone-on-referenced-issue.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable "GITHUB_TOKEN"
+checkForVariable "PULL_REQUEST_NUMBER"
+
+PULL_REQUEST_COMMITS=$(curl -s \
+  -H "Accept: application/vnd.github+json" \
+  -H "Authorization: Bearer $GITHUB_TOKEN"\
+  https://api.github.com/repos/checkstyle/checkstyle/pulls/"$PULL_REQUEST_NUMBER"/commits)
+PULL_REQUEST_LAST_COMMIT_MESSAGE=$(echo "$PULL_REQUEST_COMMITS" | jq -r .[0].commit.message)
```

Comment:
It looks like it will be easy to get array of commit messages. To be 100% precise.

Reply:
there suppose to be loop over issue numbers to put milestone for all issues in PR.

Reply:
Done.

Success job: https://github.com/stoyanK7/checkstyle/actions/runs/4305353155/jobs/7507667255
PR: https://github.com/stoyanK7/checkstyle/pull/55
Issue 1: https://github.com/stoyanK7/checkstyle/issues/36
Issue 2: https://github.com/stoyanK7/checkstyle/issues/54